src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -254,7 +254,7 @@ public FFMpegVideo(PmsConfiguration configuration) {
 					}
 				}
 			} else if (params.sid.getType().isPicture()) {
-				if (params.sid.getId() < 100) {
+				if (params.sid.isEmbedded()) {
```

Comment:
@Nadahar  I can confirm this fixed the issue #1434

Reply:
@valib That's great news 👍